tpi/terraform.py

```diff
@@ -131,3 +131,10 @@ def _shell_default(
         port = f":{port}" if port is not None else ""
         cmd.append(f"{user}{host}{port}")
         run(cmd)
+
+    def state_mv(self, source, destination, **kwargs):
+        """Retain an existing remote object but track it as a
+        different resource instance address.
+        """
+        assert source and destination
+        self.tf.cmd("state mv", source, destination)
```

Comment:
great :sweat_smile: thanks to #17 doesn't this function now become unnecessary?

@pmrowla WDYT?